tests/validation/tests/v3_api/test_node_template.py

```diff
@@ -0,0 +1,246 @@
+import os
+import pytest
+from rancher import ApiError
+from .common import *  # NOQA
+from .test_rke_cluster_provisioning import rke_config
+from .test_rke_cluster_provisioning import random_node_name
+from .test_rke_cluster_provisioning import create_and_validate_cluster
+
+DO_ACCESSKEY = os.environ.get('DO_ACCESSKEY', "None")
+engine_install_url = "https://releases.rancher.com/install-docker/19.03.sh"
+client = None
+admin_client = None
+user1_client = None
```

Comment:
Instead of declaring this globally like this
```
client = None
admin_client = None
user1_client = None
```
We could have something like
```
user_clients = {"admin": None, "standard_user_1": None, "standard_user_2": None}
```
The names of the variables will be more apt and we can declare as a single dictionary ?

Reply:
Adressed in changes. Clients have been refactored.